### ISSUE TITLE
Option to set subtitles as default when publishing video

### ISSUE THREAD

**Issue by forestrage** [Chocobozzz/PeerTube#3685]
**Describe the problem to be solved**

I have a video that is in multiple languages. Often people don't notice that they should activate subtitles and therefore miss content. 

**Describe the solution you would like:**

I would love the possibility to set a subtitle as default so that when people open the page, there are automatically subtitles on the video. 

**Describe alternatives you have considered**  

Even more useful would be if as mentioned in another feature request ( https://github.com/Chocobozzz/PeerTube/issues/2992 ) there would be different interfaces for each language and default subtitles could also be different for each language, similar to how it currently is on youtube. But I realise this is probably a massive job so already the option just to choose one default subtitle would be very helpful.

**Comment by forestrage** [comment]
Just remembered a slightly related issue: I want to embed my video on my wordpress site. When I choose the option to auto-select the subtitle, it only appears as the link. However when I just have the link without auto-select subtitle, it appears as the video.
Still to test if the problem is the same on our social media channels.

**Comment by forestrage** [comment]
Just tested on twitter and the link with subtitles does show up as a video, so maybe its an issue with wordpress.

**Comment by WonderingDane** [comment]
Great idea. At a minimum it should show a "CC" icon on the player, so that users can easily see if subtitles is available.

**Comment by rigelk** [comment]
@WonderingDane CC is not recognizable by non-english speakers though ; another icon needs to be found.

**Comment by WonderingDane** [comment]
@rigelk, very true, it was just an example.
![CCicon](https://user-images.githubusercontent.com/18703455/106759586-34cac000-6633-11eb-8951-a90c42b14369.PNG)
It could be an icon like this.

**Comment by Poslovitch** [comment]
Isn't this a (better) duplicate of #2379 ?

**Comment by NoMoreCRAPTion** [comment]
> @WonderingDane CC is not recognizable by non-english speakers though ; another icon needs to be found.

Yes, [CC] a North American thing specifically for deaf and of hard-of-hearing thing. The only way to make that work from a design perspective is to change the GUI based on viewer's location.

Everyone else uses the horizontal lines; or if it's for a deaf audience, then "SDH" aka "Subtitle for Deaf and Hard of Hearing". Icons vary by country. The icon in Australia doesn't even look the same as the icon in Finland: 
![image](https://user-images.githubusercontent.com/95659233/145446052-bd95774a-2b9f-471e-a3c1-0383c36555ea.png)
 
(Swedish version for Swedish deaf populations in Finland is the same except "Svenska")

To an English-speaker, they wouldn't know what "T" beside the subtitle icon means since that doesn't exist in UK or any other English-speaking countries.

Source: ["Accessibility in Yle’s services"](https://yle.fi/aihe/s/about-yle/accessibility-in-yles-services)

Think in India, it's just "SLS" or "Same-Language Subtitles" when it comes to indicating whether or not the subtitle is translated or subtitle is for deaf audience.

**Comment by Chocobozzz** [comment]
Hello,

I'm closing this issue now https://github.com/Chocobozzz/PeerTube/issues/2379 is implemented
